@nav-frontend/react-link/src/Link.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { forwardRef } from "react";
+import cl from "classnames";
+import "@nav-frontend/link-styles";
+
+export interface LinkProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * Link anchor should direct to
+   */
+  href: string;
+  /**
+   * User defined classname
+   */
+  className?: string;
+}
+
+const Link = forwardRef<HTMLAnchorElement, LinkProps>(
+  ({ children, className, href, ...rest }, ref) => {
+    return (
+      <a
+        ref={ref}
+        href={href}
+        className={cl("navds-link", className)}
```

Comment:
Hva tenker du om å droppe denne react-komponenten? Det eneste den gjør er vel å legge på `navds-link`-klassen på en `<a>`?

Reply:
Button-pakka vår er jo også bare klasser-egentlig. Kanskje det er like greit å bare ha det? Det gjør jo noen type endringer mye enklere.

Reply:
Interessant tanke, vi kan på mange måte se på CSS en vår som selve implementasjonen av designsystemet, og da la eks React, web-components være noe som bare implementerer designsystemet og gjør det enklere for bruker. 

Jeg tenker vi beholder komponenter som eks Lenker, chevron og button, men at vi er ekstra sikre på at CSS-en er enkel/logisk å bruke uten komponentene. Å ikke minst godt dokumentert hvordan man da bruker CSS-en på vanlig HTML, samt hvordan sikre god UU.